fretboard/src/main/java/mozilla/components/service/fretboard/HttpClient.kt

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fretboard
+
+import java.net.URL
+
+/**
+ * Represents an http client, used to
+ * make it easy to swap implementations
+ * as needed
+ */
+interface HttpClient {
```

Comment:
This is probably a bit too fine-grained. I think for now we can fold this into our KintoExperimentSource module. For now we won't need to use the same Source with different clients - Especially as Kinto only speaks HTTP.

Reply:
Ok, the reason I added that interface was in order to let the user of the library swap http client implementations (okhttp / httpurlconnection), will fold this to the KintoExperimentSource module.

Reply:
You could still keep that interface + implementation in the kinto-module and just set the httpurlconnection one as default:

```Kotlin
class KintoExperimentSource(
   client: HttpClient = HttpUrlConnectionClient(),
   ...
) {
...
}
```

But we may not need to expose it as separate module/dependency.

Reply:
Ok, looks good. I have pushed another commit with it now